diagnostics_tools/include/diagnostic_tools/periodic_event_status.h

```diff
@@ -66,9 +66,25 @@ class PeriodicEventStatusParams final {
 
   const Diagnostic &stale_diagnostic() const { return stale_diagnostic_; }
 
+  PeriodicEventStatusParams &last_cycle_period_avg_window(double last_cycle_period_avg_window) {
+    last_cycle_period_avg_window_ = last_cycle_period_avg_window;
+    return *this;
+  }
+
+  double last_cycle_period_avg_window() const {return last_cycle_period_avg_window_;}
+
+    PeriodicEventStatusParams &historic_period_avg_window(double historic_period_avg_window) {
+    historic_period_avg_window_ = historic_period_avg_window;
+    return *this;
+  }
+  
+  double historic_period_avg_window() const {return historic_period_avg_window_;}
+
  private:
   double min_acceptable_period_{-1};
   double max_acceptable_period_{5};
+  double last_cycle_period_avg_window_{1};
```

Comment:
@patapufate nit: it seems to me that `last_cycle` and `historic` are no longer adequate names. Perhaps `short_term` and `long_term`, respectively? I'm open to other alternatives.

Reply:
Changed.